README.md

```diff
@@ -75,6 +64,65 @@ The unit tests have been implemented by sending a POST request to GitHub and obs
 One test case send valid API data and expects a valid response, another sends invalid API data and
 expects an invalid response.
 
+## API Documentation
+
+### POST requests
+`/push-events`
+
+#### Description
+Pushes an event to the server
```

Comment:
This is not correct, the endpoint itself is not pushing anything. You could say something like "endpoint that accepts push events from GitHub webhooks. Or something similar.